src/brogue/Monsters.c

```diff
@@ -850,7 +846,7 @@ creature *spawnHorde(short hordeID, short x, short y, unsigned long forbiddenFla
         leader->info.intrinsicLightType = SACRIFICE_MARK_LIGHT;
     }
 
-    if (BROGUE_VERSION_ATLEAST(1,9,3) && (theHorde->flags & HORDE_MACHINE_THIEF)) {
+    if ((theHorde->flags & HORDE_MACHINE_THIEF)) {
```

Comment:
nit: nested parens

Reply:
good spot, though idm double-nesting for bitwise ops